src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/UIElement.cs

```diff
@@ -1403,8 +1403,8 @@ private static void RenderTransform_Changed(DependencyObject d, DependencyProper
         private static bool IsRenderTransformOriginValid(object value)
         {
             Point v = (Point)value;
-            return (    (!DoubleUtil.IsNaN(v.X) && !Double.IsPositiveInfinity(v.X) && !Double.IsNegativeInfinity(v.X))
-                     && (!DoubleUtil.IsNaN(v.Y) && !Double.IsPositiveInfinity(v.Y) && !Double.IsNegativeInfinity(v.Y)));
+            return (    (!double.IsNaN(v.X) && !Double.IsPositiveInfinity(v.X) && !Double.IsNegativeInfinity(v.X))
```

Comment:
These checks could use `double.IsNormal` instead of IsNan, IsPositiveInfinity, and IsNegativeInfinity.
Same on other places.

But this API isn't available on .NET Full, only starting at .NET Core 2.1 and .NET Standard 2.1.

Instead of IsPositiveInfinity and IsNegativeInfinity use just `double.IsInfinity`.

Reply:
I would implement it in a separate PR. 
This one just mechanically replaces one call with another.

Reply:
Makes sense.